pkg/artifactory/resource_artifactory_remote_repository.go

```diff
@@ -239,6 +239,30 @@ func resourceArtifactoryRemoteRepository() *schema.Resource {
 				Optional: true,
 				Default:  "",
 			},
+			"nuget": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				MaxItems: 1,
```

Comment:
Also should add minItems == 1. This is to prevent an empty nuget block in the config.